content/sips/sip-311.md

```diff
@@ -0,0 +1,100 @@
+
+## Technical Specification
+
+<!--The specification should describe the syntax and semantics of any new feature, there are five sections
+1. Overview
+2. Rationale
+3. Technical Specification
+4. Test Cases
+5. Configurable Values
+-->
+
+### Overview
+
+<!--This is a high level overview of *how* the SIP will solve the problem. The overview should clearly describe how the new feature will be implemented.-->
+
+Teleporters involve adding `transferCrossChain` and `ccipReceive` functions to the snxUSD token with appropriate validations and fee capture functionality.
+
+### Rationale
+
+<!--This is where you explain the reasoning behind how you propose to solve the problem. Why did you propose to implement the change in this way, what were the considerations and trade-offs. The rationale fleshes out what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.-->
+
+We propose making the transfer function available on the snxUSD proxy, rather than the core system proxy. This is an extension of the ERC-20 interface (which already essentially includes a "same-chain transfer" function).
+
+This proposal entails collecting fees in ETH, rather than snxUSD, for the following reasons:
+
+- Users will always receive the same amount of snxUSD that they have transferred.
+- This is the most efficient method to collect fees, as no additional token transfers are involved in the transaction.
+- Users are already paying ETH for gas fees to execute the transaction.
+
+Future SIPs could involve additional fee payment methods (such as snxUSD), alternative fee structures, and circuit-breaking functionality for added security.
+
+### Technical Specification
+
+<!--The technical specification should outline the public API of the changes proposed. That is, changes to any of the interfaces Synthetix currently exposes or the creations of new ones.-->
+
+Implementation entails two functions on the snxUSD token module:
+
+#### `transferCrossChain(uint chainId, address recipient, uint amount)`
+
+- Confirm that the sender has a sufficient balance of snxUSD to transfer.
+- Determine the fee amount in ETH by taking an SCCP-specified percentage of the amount transferred (plus any SCCP-specified flat fee) and denominate that amount in ETH based on a price oracle.
+- Transfer the fee into the contract and notify a rewards distributor of the collected fees.
+- Encode the `recipient` and `amount` and call `ccipSend`.
+- Burn the amount of snxUSD transferred.
+
+#### `ccipReceive(Any2EVMMessage calldata message)`
```

Comment:
might be worthwhile to mention here that `ccipReceive` is implementing a function from the CCIP specification, we aren't defining a new function here